fontbe/src/font.rs

```diff
@@ -0,0 +1,41 @@
+//! Merge tables into a font
+
+use fontdrasil::orchestration::Work;
+use read_fonts::types::Tag;
+use write_fonts::FontBuilder;
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Bytes, Context, WorkId},
+};
+
+struct FontWork {}
+
+pub fn create_font_work() -> Box<BeWork> {
+    Box::new(FontWork {})
+}
+
+// TODO the tables have the tag, but I'm not sure how to access it
+const TABLES_TO_MERGE: &[(WorkId, Tag)] = &[
+    (WorkId::Cmap, Tag::new(b"cmap")),
+    (WorkId::Hmtx, Tag::new(b"hmtx")),
+    (WorkId::Glyf, Tag::new(b"glyf")),
+    (WorkId::Loca, Tag::new(b"loca")),
```

Comment:
needless to say, hmtx won't work without hhea, and loca (thus glyf) won't work without head (likely other tables too depend on head too), but those are easy if you just want something that works and don't care about the particular values in it. OS/2 might be needed for windows I think (just use the same vertical metrics values as hhea).

Reply:
oh, name table seems to be required by OTS (I just tried to strip it from a font and it complained)

Reply:
Yup. I think I want to merge this then add more tables as we progress through #138.

Reply:
To paraphrase me, this puts the pattern to create a working font in place. It doesn't yet fully do it due to missing tables. My current expectation is if we hit the set identified in #138 we're good.